src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -260,7 +261,7 @@ public void UseShellExecute_GetSetWindows_Success_Netcore()
 
         [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
We agreed to default to false on .NET Core for Windows, and we're already false for Unix. That seems appropriate (consisetncy across Core).
I caused confusion by commenting at the wrong place.